cms/static/js/certificates/spec/views/certificate_preview_spec.js

```diff
@@ -7,7 +7,8 @@ define([ // jshint ignore:line
     'js/certificates/views/certificate_preview',
     'common/js/spec_helpers/template_helpers',
     'common/js/spec_helpers/view_helpers',
-    'common/js/spec_helpers/ajax_helpers'
+    'common/js/spec_helpers/ajax_helpers',
+    'jasmine-stealth'
```

Comment:
Note to reviewers: there are several unit tests that did not properly declare all their dependencies. This test was passing when running the complete suite (because some earlier test required jasmine-stealth), but failed when run individually. There will be similar changes below.